lib/trento/application/projectors/database_projector.ex

```diff
@@ -159,6 +160,25 @@ defmodule Trento.DatabaseProjector do
     end
   )
 
+  project(
+    %DatabaseInstanceDeregistered{
+      instance_number: instance_number,
+      host_id: host_id,
+      sap_system_id: sap_system_id,
+      deregistered_at: _deregistered_at
+    },
+    fn multi ->
+      deregistered_instance =
+        Repo.get_by(DatabaseInstanceReadModel,
```

Comment:
I don't think you need a `get_by`. Just creating the read model with the primary keys should be enough.
https://hexdocs.pm/ecto/Ecto.Repo.html#c:delete/2